### PR TITLE
cash-1313 exp initial setup, couple kinks to workout

### PR DESCRIPTION
**Pull request by BoulderBrains**

### CODE REVIEW

File: src/pages/Homepage/HeroSlideshow.vue [hunk 1]
```diff
@@ -50,7 +50,32 @@
 				&nbsp;
 			</div>
 		</div>
-		<div class="action-button-wrapper">
+		<div
```
Comment (mcstover) [inline src/pages/Homepage/HeroSlideshow.vue]: @BoulderBrains you shouldn't need to duplicate this markup. You can just drop a span after the copy in each button `Sign up<span v-if="showDoubleArrowButton"> >></span>`

File: src/pages/Homepage/HeroSlideshow.vue [hunk 2]
```diff
@@ -82,6 +107,7 @@ const mgPromoImageRequire = require.context('@/assets/images/mg-hero-slideshow/'
 
 export default {
 	name: 'HeroSlideshow',
+	// I need to return the props.doubleArrowButtonExp like the following example is doing for mgPromoExp
```
Comment (mcstover) [inline src/pages/Homepage/HeroSlideshow.vue]: For this you can add some logic above the return statement to determine which of your 4 cases should be the cacheKey. Make sure to give a unique key for your 2 new cases...then just return you final key.

File: src/pages/Homepage/Homepage.vue [hunk 3]
```diff
@@ -41,6 +51,13 @@ export default {
 				this.mgPromoExp.version === 'shown' ? 'b' : 'a'
 			);
 		}
+		if (this.doubleArrowButtonExp.version !== null) {
+			this.$$kvTrackEvent(
```
Comment (mcstover) [inline src/pages/Homepage/Homepage.vue]: 1 $ no 2 $$

Comment (BoulderBrains) [reply src/pages/Homepage/Homepage.vue]: Mo money, Mo problems.

File: src/pages/Homepage/HeroSlideshow.vue (hunk 2 above)
Comment (BoulderBrains) [reply src/pages/Homepage/HeroSlideshow.vue]: 👍

**Pull request merged**